pkg/config/msp_test.go

```
@@ -1314,3 +1258,59 @@ func TestRevokeCertificateFromMSPFailure(t *testing.T) {
 		})
 	}
 }
+
+func baseMSP() MSP {
+	certBlock, _ := pem.Decode([]byte(dummyCert))
```

Why are we ignoring the error? Even in a test, if we expect it to work, we should assert.